ui/components/ui/box/box.stories.js

```diff
@@ -217,27 +250,103 @@ export const Margin = () => {
   );
 };
 
-export const Padding = () => {
+export const Padding = (args) => {
   return (
     <Box borderColor={COLORS.BORDER_MUTED}>
       <Box
         padding={4}
         backgroundColor={COLORS.BACKGROUND_ALTERNATIVE}
         borderColor={COLORS.BORDER_MUTED}
+        {...args}
       >
         Static padding
       </Box>
       <Box
         padding={[4, 8, 12]}
         backgroundColor={COLORS.BACKGROUND_ALTERNATIVE}
         borderColor={COLORS.BORDER_MUTED}
+        {...args}
       >
         Responsive padding changes based on breakpoint
       </Box>
     </Box>
   );
 };
 
+export const Color = (args) => {
```

Comment:
Moving order of `Color` story to above `backgroundColor`


<img width="244" alt="Screen Shot 2022-08-11 at 4 24 00 PM" src="https://user-images.githubusercontent.com/8112138/184258560-070f0bb6-ab0c-4808-a226-dc085c934e4d.png">